test-suite/Network/CloudSeeder/Test/Stubs.hs

```diff
@@ -39,10 +39,10 @@ instance Monad m => MonadCLI (ArgumentsT m) where
   getOptions pSpecs = ArgumentsT $ do
     input <- ask
     let x = execParserPure defaultPrefs (parseOptions pSpecs) input
-    return $ fromJust $ getParseResult x
+    return $ unsafePerformIO $ handleParseResult x
```

Comment:
😱 Could you describe what information `handleParseResult` provides that `getParseResult` doesn’t?